src/Response/Response.php

```diff
@@ -31,12 +31,7 @@ public function __construct(ResponseInterface $response)
         $this->response = $response;
     }
 
-    public function __set(string $name, $value): void
-    {
-        throw new LogicException('Cannot set proporties on responses.');
-    }
-
-    public function __get(string $name)
+    public function get(string $name)
```

Comment:
I would prefer to have DocBlocks here with the more generic methods `get, `has` etc.  compared to `setLimit`.